File: 003/fizzbuzz.c
#include <stdio.h>

#define STOP_AT 1000000
#define FIZZ 3
#define BUZZ 5

void display(int);

int main(void)
{
  for (int i=0; i<STOP_AT; i++)
    display(i);
}

void display(int i)
{
  if (i % FIZZ == 0)
    printf("FIZZ\n");
  else if (i % BUZZ == 0)
    printf("BUZZ\n");
  else
    printf("%d\n", i);
}

File: 003/fizzbuzz.py
def main():
    for i in range(1000000):
        display(i)


def display(i):
    if not i % 3:
        print("Fizz")
    elif not i % 5:
        print("Buzz")
    else:
        print(i)


if __name__ == '__main__':
    main();
